nerdlandbot/helpers/constants.py

```diff
@@ -16,3 +16,14 @@
 
 POLL_MAX_TIMEOUT = 60 #minutes
 POLL_DEFAULT_TIMEOUT = 5 #minutes
+
+# the space_launches constants used
+THE_SPACE_DEVS_BASE_URL = 'https://ll.thespacedevs.com'
+THE_SPACE_DEVS_VERSION = '2.0.0'
+THE_SPACE_DEVS_UPCOMING_LAUNCH_RESOURCE = ['launch','upcoming']
+THE_SPACE_DEVS_LIMIT_TO_10_RESULTS = '?limit=10&offset=0'
+THE_SPACE_DEVS_LOCAL_TEST_SERVER_BASE_URL = 'http://localhost:8000'
```

Comment:
Is this constant necessary in the production bot?

Reply:
See above, but I can clean it up a bit for production actually.